sql/pre_install/tables.sql

```diff
@@ -133,23 +133,26 @@ SELECT pg_catalog.pg_extension_config_dump(pg_get_serial_sequence('_timescaledb_
 -- the chunk's hypercube. Tuples that fall within the chunk's
 -- hypercube are stored in the chunk's data table, as given by
 -- 'schema_name' and 'table_name'.
+CREATE SEQUENCE IF NOT EXISTS _timescaledb_catalog.chunk_id_seq MINVALUE 1;
+
 CREATE TABLE IF NOT EXISTS _timescaledb_catalog.chunk (
-  id serial NOT NULL PRIMARY KEY,
+  id integer PRIMARY KEY DEFAULT nextval('_timescaledb_catalog.chunk_id_seq'),
```

Comment:
Not sure *why* you did this change. Could you explain? Maybe in the PR message as well.

Reply:
serial creates a sequence automatically. By creating the sequence ourselves, we control its naming and useful for the update script as well. (You'll see the same behavior for timescaledb_catalog.hypertable table)